package.json

```diff
@@ -1,6 +1,6 @@
 {
-	"name": "@versatile-ai/sequelize-paper-trail",
-	"version": "3.0.7",
+	"name": "@vnatures/sequelize-paper-trail",
+	"version": "3.1.0",
```

Comment:
maybe change version to 4.0.0 as it's a potentially breaking change once you move to a new sequelize major